RandomObjects.py

```diff
@@ -133,7 +133,10 @@ def generateSeq(self):
         # robot's up most initial y position
         robot_up = self.height / 2 + robot_radius / tile_size + 1
         goodLoc = False
-        while not goodLoc:
+        numLoops= 0
+        maxLoops= 10
+        while numLoops <= maxLoops and not goodLoc:
+            numLoops+= 1
```

Comment:
fix this so that the loop terminates on its own without encountering any edge cases

Reply:
Yep, a hacky fix we need to resolve